docs/10-01-howto-feature-try-runtime.md

```diff
@@ -0,0 +1,319 @@
+## Sub-Command Execution Status
+
+| Test ID | Sub-Command               | Status |
+| ------- | ------------------------- | ------ |
+| 1       | create-snapshot           | OK     |
+| 2       | on-runtime-upgrade (live) | OK     |
+| 3       | on-runtime-upgrade (snap) | OK     |
+| 4       | execute-block (live)      | OK     |
+| 5       | execute-block (snap)      | OK     |
+| 6       | offchain-worker (live)    | OK     |
+| 7       | offchain-worker (snap)    | OK     |
+| 8       | follow-chain              | OK     |
+| 9       | fast-forward (live)       | KO     |
+| 10      | fast-forward (snap)       | KO     |
+
+Tests with ID 9 and 10 have encountered failures, with the error message "ERROR
+main runtime: panicked at 'Timestamp slot must match `CurrentSlot`'". The
+underlying cause of this issue is currently unknown. To facilitate the
+resolution and ensure progress, a support ticket has been raised and submitted
+to the substrate. For more information, please visit
+[here](https://substrate.stackexchange.com/questions/9024/substrate-try-runtime-sub-command-fast-forward-error-main-runtime-pa).
+
+## How to Build and Run under Local Testnet
+
+To build and run under the local testnet, follow the steps below:
+
+- Create the release binary and launch the local testnet using the script
+  `infra/local-testnet/run-cfg2.sh`:
+
+```bash
+cargo build --release --bin madara
+```
+
+- Execute the `try-runtime` subcommands against the local testnet:
+
+```bash
+cargo run --release --features=try-runtime --bin madara -- try-runtime --help
+```
+
+This will provide you with a list of available `try-runtime` subcommands:
+
+- `on-runtime-upgrade`: Execute the migrations of the given runtime.
+- `execute-block`: Executes the given block against some state.
+- `offchain-worker`: Executes the offchain worker hooks of a given block against
+  some state.
+- `follow-chain`: Follow the given chain's finalized blocks and apply all of its
+  extrinsics.
+- `fast-forward`: Produce a series of empty, consecutive blocks and execute them
+  one-by-one.
+- `create-snapshot`: Create a new snapshot file.
+- `help`: Print this message or the help of the given subcommand(s).
+
+Feel free to refer to these instructions when building and running under the
+local testnet.
+
+================================================================
+
+## create-snapshot
+
+> Step-01 | create snapshot from sharingan testnet
+
+```bash
+export RPC_WS_URL="ws://127.0.0.1:9931" && \
```

Comment:
9931 ?

Reply:
Here is the setup for the local testnet and port mapping, where `9931` is used as the RPC endpoint from `boot-node-a`.

| node            | p2p port | RPC port | prometheus port |
| --------------- | -------- | -------- | --------------- |
| boot-node-a     | 30331    | 9931     | 9611            |
| validator-node-a| 30332    | 9932     | 9621            |
| validator-node-b| 30333    | 9933     | 9631            |
| validator-node-c| 30334    | 9934     | 9641            |
| validator-node-d| 30335    | 9935     | 9651            |
| validator-node-e| 30336    | 9936     | 9661            |
| full-node-a     | 30344    | 9944     | -               |

Reply:
> Why define a new chain spec ? cfg2 ?
We shouldn't do this. @shamb0 please add the generation of the spec on the script to a tmp folder and use it directly from there. We will drop the chainspecs on our repo in the near future (they'll be retrieved from an URL)

Reply:
Sure, @d-roak, that sounds good. I will make the modifications as you suggested.